src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -216,97 +325,5461 @@ static ColliderJntSphInit D_80B7A888 = {
+            } else {
+                sp10C.x = -10.0f;
+            }
+            sp10C.y = 0.0f;
+            sp10C.z = 0.0f;
+            Matrix_RotateY(sLureRot.y, MTXMODE_NEW);
+            Matrix_MultVec3f(&sp10C, &sp100);
+
+            this->unk_1B4.x = sLurePos.x + sp100.x;
+            this->unk_1B4.z = sLurePos.z + sp100.z;
+
+            if (D_80B7E0B6 == 2) {
+                this->unk_1B4.y = sLurePos.y;
+            } else if (this->unk_150 == 0) {
+                this->unk_1B4.y = sLurePos.y - 15.0f;
+            } else {
+                this->unk_1B4.y = sLurePos.y - 5.0f;
+            }
+
+            if (this->unk_1B4.y <= this->actor.floorHeight) {
+                this->unk_1B4.y = this->actor.floorHeight + 3.0f;
+            }
+
+            if ((D_80B7E0B6 != 2) && (this->unk_1B4.y < this->actor.world.pos.y)) {
+                Math_ApproachF(&this->actor.world.pos.y, this->unk_1B4.y, 0.1f,
+                               (this->actor.world.pos.y - this->unk_1B4.y) * 0.1f);
+            }
+
+            Math_ApproachF(&this->unk_1B0, 8192.0f, 1.0f, (KREG(16) * 128) + 384.0f);
+            if (CHECK_BTN_ALL(input->press.button, BTN_A)) {
+                this->unk_1A8 += 0.005f;
+            }
+
+            if (D_80B7E120 != 0) {
+                if (D_80B7E120 == 1) {
+                    this->unk_1A8 += 0.01f;
+                } else {
+                    this->unk_1A8 += 0.05f;
+                }
+                D_80B7E120 = 0;
+            }
+
+            if (CHECK_BTN_ALL(input->press.button, BTN_B)) {
+                this->unk_1A8 += 0.008f;
+            }
+
+            if (sp124 < ((this->unk_1AC * 0.5f) + 20.0f)) {
+                if (this->unk_15E == 0) {
+                    this->unk_190 = 1.0f;
+                    this->unk_194 = 500.0f;
+                    this->unk_17A[0] = (s16)Rand_ZeroFloat(10.0f) + 2;
+                }
+                Math_ApproachF(&this->actor.speedXZ, -0.2f, 1.0f, 0.1f);
+                this->unk_15E = 1;
+            } else {
+                if (this->unk_15E != 0) {
+                    this->unk_190 = 1.0f;
+                    this->unk_1B0 = 0.0f;
+                    this->unk_194 = 3000.0f;
+                }
+                Math_ApproachF(&this->actor.speedXZ, 3.0f, 1.0f, 0.15f);
+                this->unk_15E = 0;
+            }
+
+            if (this->unk_1AC >= 60.0f) {
+                sp120 = 0.3f;
+            } else if (this->unk_1AC >= 45.0f) {
+                sp120 = 0.6f;
+            } else {
+                sp120 = 1.0f;
+            }
+
+            if ((gSaveContext.dayTime >= 0xB555) && (gSaveContext.dayTime <= 0xCAAA)) {
+                sp120 *= 1.75f;
+            } else if ((gSaveContext.dayTime >= 0x3555) && (gSaveContext.dayTime <= 0x4AAA)) {
+                sp120 *= 1.5f;
+            } else if (D_80B7E076 != 0) {
+                sp120 *= 1.5f;
+            } else if ((u8)D_80B7A650 != 0) {
+                sp120 *= 3.0f;
```

Comment:
Based on past documentation this is probably the multiplier and can be named as such.